### PR TITLE
Updated form elements

### PR DESCRIPTION
**Pull request by Pallavi-PH**
1. Updated form elements
2. Removed green button as per discussion with Victoria

Screenshot 2020-09-21 at 5 21 18 PM
Screenshot 2020-09-21 at 5 21 28 PM
Screenshot 2020-09-21 at 5 21 36 PM
Screenshot 2020-09-21 at 5 21 45 PM
Screenshot 2020-09-21 at 5 21 52 PM
Screenshot 2020-09-21 at 5 22 53 PM
Screenshot 2020-09-21 at 5 24 40 PM
Screenshot 2020-09-21 at 5 25 05 PM

### CODE REVIEW

File: docs/products/forms.mdx [hunk 1]
```diff
@@ -92,8 +112,12 @@ Set heights using classes like .form-control-lg and .form-control-sm.
 ## Radio
 <Playground>
 <div class="custom-control custom-radio">
-  <input type="radio" id="customRadio1" name="customRadio" class="custom-control-input"/>
-  <label class="custom-control-label" for="customRadio1">Label</label>
+  <input type="radio" id="customRadio1" name="customRadio" class="custom-control-input" checked/>
```
Comment (is-ashish) [inline docs/products/forms.mdx:115]: @Pallavi-PH Checked `Checkbox`  color will be primary or green here ? many places seen input type checkbox checked have green theme.

Comment (Pallavi-PH) [reply docs/products/forms.mdx:115]: It will be green, as in the screenshot

Comment (is-ashish) [reply docs/products/forms.mdx:115]: Ok ,Thanks

**Review by is-ashish — APPROVED** [review]

**Review by avishnu — APPROVED** [review]
LGTM

**Pull request merged**